### ISSUE TITLE
[stable/grafana] subjects[0].namespace: Required value

### ISSUE THREAD

**Issue by remster** [helm/charts#6160]
**Version of Helm and Kubernetes**:
[remek][~/.../xxx][remek*]$ helm version
Client: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.9.1", GitCommit:"20adb27c7c5868466912eebdf6664e7390ebe710", GitTreeState:"clean"}
[remek][~/.../xxx][remek*]$ kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.4", GitCommit:"5ca598b4ba5abb89bb773071ce452e33fb66339d", GitTreeState:"clean", BuildDate:"2018-06-06T08:13:03Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}

**Which chart**:
stable/grafana

**What happened**:
```
[remek*]$ helm install stable/grafana
Error: release masked-badger failed: ClusterRoleBinding.rbac.authorization.k8s.io "masked-badger-grafana-clusterrolebinding" is invalid: subjects[0].namespace: Required value
```
**What you expected to happen**:
No error

**How to reproduce it** (as minimally and precisely as possible):
`helm install stable/grafana`

**Anything else we need to know**:
Seemingly caused by this: https://github.com/kubernetes/charts/commit/660dc153d14f1fccfe1c92e3c9059dc219c933d7#diff-8ebb7754fa875df6cca6c9f90c9a5e72R18

**Comment by remster** [comment]
Well. It's `Release.Namespace` not `Release.namespace`

**Comment by amartorelli** [comment]
Noticed the same problem this morning when installing Grafana. Tried installing it on Friday and worked perfectly so that commit seems to be the problem. +1

**Comment by davidkarlsen** [comment]
@remster 
can be /close'd 
fixed by PR #6158
